primefaces-showcase/src/main/webapp/accessibility.xhtml

```diff
@@ -1,8 +1,233 @@
 <ui:composition xmlns="http://www.w3.org/1999/xhtml" xmlns:h="http://java.sun.com/jsf/html" xmlns:f="http://java.sun.com/jsf/core"
-    xmlns:ui="http://java.sun.com/jsf/facelets" xmlns:p="http://primefaces.org/ui" template="/WEB-INF/template.xhtml">
+                xmlns:ui="http://java.sun.com/jsf/facelets" xmlns:p="http://primefaces.org/ui" template="/WEB-INF/template.xhtml">
 
-    <ui:define name="title">
-        Accessibility
+    <ui:define name="content">
+        <div class="content-section introduction">
+            <div class="feature-intro">
+                <h1>Accessibility</h1>
+                <p>An introduction to accessibility and how it translates to JSF UI Components.</p>
+            </div>
+        </div>
+
+        <div class="content-section documentation">
+            <p class="line-height-3 bg-indigo-600 text-white p-3 text-lg" style="border-radius:10px">Accessibility is a major concern of the Prime UI libraries and PrimeFaces is no exception. <a href="https://www.primetek.com.tr" class="text-white" target="_blank">PrimeTek</a> teams have initiated a significant process to review and enhance the accessibility features of the components. This guide
+                documents the foundation of the general guidelines that PrimeFaces will follow and each component documentation will have a separate <b>Accessibility</b> section that states the keyboard support, screen reader compatibility, the implementation details along
+                with tips to achieve WCAG compliancy. This work has been initiated in Q2 2022 and planned to be completed with PrimeFaces 13.0.0 in Q1 2013. PrimeVue acts as the reference implementation which then will be ported to PrimeFaces, PrimeNG and PrimeReact.
```

Comment:
Is PrimeFaces 13.0.0 in Q1 2013 realistic?

Reply:
you mean 2023?  And I wouldn't put a quarter or release on it just say 2023?

Reply:
ups. yep - 2023